### ISSUE TITLE
Feature-Request: Formatting option

### ISSUE THREAD

**Issue by samuelT2** [KamasamaK/vscode-cfml#19]
Hi there

Very useful plugin, thank you.
As I'm working a lot with legacy code which is poorly formatted, I'd love to see a format command (similiar to the inbuild shift+alt+f command). Is this something which is already on your roadmap?

Best
Sam

**Comment by mgit7** [comment]
Yeah, I would also like to see so auto-formatting option for CFML in VSCode. It really is a pain to have to format it manually.
